custom/echis_reports/ucr/data_sources/household_characteristics_data_source.json

```diff
@@ -0,0 +1,501 @@
+{
+   "domain":"fmoh-echis-staging",
+   "es_index_settings":{
+      "doc_type":"ElasticSearchIndexSettings",
+      "refresh_interval":"5s",
+      "number_of_shards":2
+   },
+   "configured_indicators":[
```

Comment:
The config that you find on the web page should be under a field called `config` in this doc. Please see an example [static data source](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/ucr/data_sources/daily_feeding_forms.json#L20).

Reply:
@sravfeyn  should I wrap the data sources like the data source you sent me .

Reply:
This is not addressed.